mkdocs.yml

```diff
@@ -36,30 +39,9 @@ markdown_extensions:
   - codehilite:
       guess_lang: false
       linenums: false
-  - def_list
-  - footnotes
-  - meta
   - smarty
-  - toc:
-      permalink: true
-  - pymdownx.betterem:
```

Comment:
Do we not use any of these? smarty was the one I knew we used.

Reply:
the theme uses them, but the theme includes them

Reply:
do we do anything in particular with smarty? ... it like converts characters or something?

Reply:
Smart quotes and dashes.

Reply:
smarty stays, because it's ours. This way our extensions section expresses the ones related to our site, and the theme expresses what it wants.